Pritunl not starting

Pritunl doesn't start with this stacktrace : 
```
pritunl    | [winter-fields-7139][2018-06-14 19:45:09,027][INFO] Generating server certificate...
pritunl    | Traceback (most recent call last):
pritunl    |   File "/usr/bin/pritunl", line 11, in <module>
pritunl    |     load_entry_point('pritunl==1.29.1651.76', 'console_scripts', 'pritunl')()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/__main__.py", line 358, in main
pritunl    |     pritunl.init_server()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/__init__.py", line 21, in init_server
pritunl    |     app.run_server()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/app.py", line 255, in run_server
pritunl    |     _run_wsgi()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/app.py", line 229, in _run_wsgi
pritunl    |     _run_server(restart)
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/app.py", line 161, in _run_server
pritunl    |     setup_server_cert()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/app.py", line 237, in setup_server_cert
pritunl    |     utils.create_server_cert()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/utils/cert.py", line 14, in create_server_cert
pritunl    |     server_cert_path, server_key_path = generate_server_cert()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/utils/cert.py", line 49, in generate_server_cert
pritunl    |     '-out', server_cert_path,
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/utils/misc.py", line 213, in check_output_logged
pritunl    | [winter-fields-7139][2018-06-14 19:45:09,052][ERROR] Popen returned error exit code
pritunl    |   cmd         = ['openssl', 'req', '-new', '-batch', '-x509', '-days', '3652', '-key', u'/tmp/pritunl_99a33a61feaa423b891fc511fce0ac66/server.key', '-out', u'/tmp/pritunl_99a33a61feaa423b891fc511fce0ac66/server.crt']
pritunl    |   return_code = 1
pritunl    | Process stderr:
pritunl    |   error, no objects specified in config file
pritunl    |   problems making Certificate Request
pritunl    | Traceback (most recent call last):
pritunl    |   File "/usr/bin/pritunl", line 11, in <module>
pritunl    |     load_entry_point('pritunl==1.29.1651.76', 'console_scripts', 'pritunl')()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/__main__.py", line 358, in main
pritunl    |     pritunl.init_server()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/__init__.py", line 21, in init_server
pritunl    |     app.run_server()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/app.py", line 255, in run_server
pritunl    |     _run_wsgi()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/app.py", line 229, in _run_wsgi
pritunl    |     _run_server(restart)
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/app.py", line 161, in _run_server
pritunl    |     setup_server_cert()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/app.py", line 237, in setup_server_cert
pritunl    |     utils.create_server_cert()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/utils/cert.py", line 14, in create_server_cert
pritunl    |     server_cert_path, server_key_path = generate_server_cert()
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/utils/cert.py", line 49, in generate_server_cert
pritunl    |     '-out', server_cert_path,
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/utils/misc.py", line 209, in check_output_logged
pritunl    |     stderr=stderrdata,
pritunl    |   File "/usr/lib/python2.7/site-packages/pritunl-1.29.1651.76-py2.7.egg/pritunl/logger/__init__.py", line 55, in error
pritunl    |     kwargs['traceback'] = traceback.format_stack()    return_code, cmd, output=stdoutdata)
pritunl    | subprocess.CalledProcessError: Command '['openssl', 'req', '-new', '-batch', '-x509', '-days', '3652', '-key', u'/tmp/pritunl_99a33a61feaa423b891fc511fce0ac66/server.key', '-out', u'/tmp/pritunl_99a33a61feaa423b891fc511fce0ac66/server.crt']' returned non-zero exit status 1
```
It seems that it can't generate ssl certificate

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM alpine:3.7
+FROM alpine:3.8
 MAINTAINER Victor Trac <victor@cloudkite.io>
 
 ENV VERSION="1.29.1705.7"
@@ -26,6 +26,9 @@ RUN wget https://github.com/pritunl/pritunl/archive/${VERSION}.tar.gz \
     && cd .. \
     && rm -rf *${VERSION}* \
     && rm -rf /tmp/* /var/cache/apk/*
+
+RUN sed -i -e '/^attributes/a prompt\t\t\t= yes' /etc/ssl/openssl.cnf
+RUN sed -i -e '/countryName_max/a countryName_value\t\t= US' /etc/ssl/openssl.cnf
 
 ADD rootfs /
 

